pytools/mpi.py

```diff
@@ -29,3 +31,184 @@ def run_with_mpi_ranks(py_script, ranks, callable_, args=(), kwargs=None):
     check_call(["mpirun", "-np", str(ranks),
         sys.executable, py_script, "--mpi-relaunch", callable_and_args],
         env=newenv)
+
+
+# {{{ MPI executors
```

Comment:
Could you link this into the documentation? (There might not be an MPI section yet, please create one. :)